src/AppComplianceAutomation/AppComplianceAutomation.Autorest/README.md

```diff
@@ -0,0 +1,84 @@
+<!-- region Generated -->
+# Az.AppComplianceAutomation
+This directory contains the PowerShell module for the AppComplianceAutomation service.
+
+---
+## Info
+- Modifiable: yes
+- Generated: all
+- Committed: yes
+- Packaged: yes
+
+---
+## Detail
+This module was primarily generated via [AutoRest](https://github.com/Azure/autorest) using the [PowerShell](https://github.com/Azure/autorest.powershell) extension.
+
+## Module Requirements
+- [Az.Accounts module](https://www.powershellgallery.com/packages/Az.Accounts/), version 2.7.5 or greater
+
+## Authentication
+AutoRest does not generate authentication code for the module. Authentication is handled via Az.Accounts by altering the HTTP payload before it is sent.
+
+## Development
+For information on how to develop for `Az.AppComplianceAutomation`, see [how-to.md](how-to.md).
+<!-- endregion -->
+
+### AutoRest Configuration
+> see https://aka.ms/autorest
+
+```yaml
+commit: 12ff37009808948b5c7ed4a0d384181471f3219f
+require:
+# readme.azure.noprofile.md is the common configuration file
+  - $(this-folder)/../../readme.azure.noprofile.md
+  - $(repo)/specification/appcomplianceautomation/resource-manager/readme.md
+try-require: 
+  - $(repo)/specification/appcomplianceautomation/resource-manager/readme.powershell.md
```

Comment:
Could you please explain why this file is needed here?

Reply:
It is not needed, removed.